renku/models/cwl/command_line_tool.py

```diff
@@ -255,7 +292,66 @@ def watch(self, client, no_output=False, outputs=None):
 
         yield tool
 
-        if repo:
+        if repo and self.explicit_outputs:
+            # TODO make sure that output paths are not reserved
+            # TODO check if outputs appear in inputs and warn the user
```

Comment:
Open a ticket or implement?

Reply:
These are already taken care of. I forgot to delete them.